model/src/pyrenew/model/hospitalizations.py

```diff
@@ -1,11 +1,8 @@
-#!/usr/bin/env/python
-# -*- coding: utf-8 -*-
-
 from collections import namedtuple
 
-from pyrenew.metaclasses import Model, RandomProcess, _assert_sample_and_rtype
-from pyrenew.models.rtinfectionsrenewal import RtInfectionsRenewalModel
-from pyrenew.processes import RtRandomWalkProcess
+from pyrenew.metaclasses import Model, RandomVariable, _assert_sample_and_rtype
```

Comment:
if we're renaming `pyrenew.models` --> `pyrenew.model`, `pyrenew.processes` --> `pyrenew.process` etc, rename `pyrenew.metaclasses`--> `pyrenew.metaclass`?